source/application/DbgHelp/DbgHelpTypes.h

```diff
@@ -210,34 +222,43 @@ class DbgHelpEnumType
 class DbgHelpPointerType
 {
 public:
-    const std::variant<DbgHelpBasicType,
-                       DbgHelpUnknownType,
-                       DbgHelpEnumType,
-                       DbgHelpUnknownUnsizedType> m_underlying_type;
+    const std::

```

Comment:
This isn't better than previous. Is there a preference option?

Reply:
Can't find such an option. Disabled with clang-format off.